policyengine_us/parameters/gov/states/co/tax/income/subtractions/military_retirement/age_threshold.yaml

```diff
@@ -1,16 +1,17 @@
 description: Colorado limits its military retirement subtraction to filers below this age.
 values:
-  2019-01-01: 55
+  2021-01-01: 54
```

Comment:
I didn't see 54 in the law. Not sure the reference goes to the right place. Can you point to where it's age<=54 rather than age<55?

Reply:
https://tax.colorado.gov/sites/tax/files/documents/Book0104_2023.pdf#page=12
![Uploading image.png…]()

Reply:
<img width="367" alt="image" src="https://github.com/PolicyEngine/policyengine-us/assets/104268340/793d0c82-0b60-45f2-a9d4-d3771ce90bd4">

Reply:
It is 54 in the tax form instruction.
<img width="366" alt="image" src="https://github.com/PolicyEngine/policyengine-us/assets/104268340/d3e84b5c-4f9f-46bf-b59c-5e4b6b4519d7">
https://tax.colorado.gov/sites/tax/files/documents/Book0104_2023.pdf#page=12

Reply:
But in the legal code, it shows "under 55 years old".
<img width="953" alt="image" src="https://github.com/PolicyEngine/policyengine-us/assets/104268340/14765a5f-6a2d-4dd0-bbad-4952c7005a3b">
https://advance.lexis.com/documentpage/?pdmfid=1000516&crid=7b09f692-5b42-4137-b2c6-b647bae08b2b&config=014FJAAyNGJkY2Y4Zi1mNjgyLTRkN2YtYmE4OS03NTYzNzYzOTg0OGEKAFBvZENhdGFsb2d592qv2Kywlf8caKqYROP5&pddocfullpath=%2fshared%2fdocument%2fstatutes-legislation%2furn%3acontentItem%3a689J-29B3-GXF6-81G9-00008-00&pdcontentcomponentid=234176&pdteaserkey=sr2&pditab=allpods&ecomp=bs65kkk&earg=sr2&prid=28da6d1b-814f-4ace-b369-4a0d0233db83

Reply:
let's go with the law and keep at 55

Reply:
Got it